src/gatsby-node.js

```diff
@@ -2,13 +2,11 @@ import { unlinkSync } from "fs"
 import path from "path"
 import LoadablePlugin from "@loadable/webpack-plugin"
 
+const isWin = process.platform === "win32";
+
 const loadableStatsName = "loadable-stats-build-javascript.json";
-const statsPath = path.join(
-  process.cwd(),
-  `/.cache/${loadableStatsName}`
-)
 
-const isWin = process.platform === "win32";
+const statsPath = _path.default.join(process.cwd(), `/.cache/${loadableStatsName}`);
```

Comment:
Where is `_path` defined?

Reply:
Oh my bad 😅, I was testing out the fix in `node_modules` where it has the `path` as `_path`, will change `_path.default` to `path`